part-configuration.tex

```diff
@@ -187,20 +187,245 @@ \section{Matching your business processes}
 \section{Administrative user}
 \label{sec-company-config-admin-user}
 
-\section{Chart of accounts}
+@@@ TODO Need to add content explaining the administrative user.
+
+\section{Chart of Accounts}
 \label{sec-company-config-coa}
 
-@@@ Should refer to the 'administration' section???
+% \chapter{Chart of accounts}
+% \label{cha-chart-of-accounts}
+
+The Chart of Accounts or \gls{CoA}\index{Chart of Accounts}\index{CoA} defines how financial reports are organized and summarized.
+This section describes the various configuration options available when creating or editing the \gls{CoA}.
+
+\subsection{Predefined Chart of Accounts}
+\label{sec-coa-predefined}
+
+LedgerSMB provides predefined Chart of Accounts\index{Chart of Accounts predefined} that can be used out of the box for many businesses.
+Most new LedgerSMB setups start with one of the predefined Chart of Accounts and add or delete as necessary to customize it for their business.
+
+There is also an option to import a Chart of Accounts. Importing is described in \secref{subsec-coa-importing}.

```

Comment:
Should we add a reference to the XSD (XML Schema Definition) file which allows people to develop their own CoA XML file?

Reply:
Added at line 208.